change_log/next/auto-focus-on-experimental-fields.yml

```diff
@@ -0,0 +1,2 @@
+New Features: "Adds `autoFocus` property to experimental input components"
+Bug Fixes: "Fixes a bug in storybook: visible semi-column removed from storybook preview page"
```

Comment:
Hey @blaky assuming `semi-column` is a typoI believe this bug was fixed and merged into master the other day as part of this commit https://github.com/Sage/carbon/pull/2225/commits/b4ba1da772eb1a91ecbdf9c0131b5c3fbf082db0? If so this line in the change_log isn't needed